### DISCUSSION TITLE
Is there a workaround for CosmoDB not fully supporting Mongo DB API v6.0

### DISCUSSION THREAD

**Discussion by kharalanova** [prisma/prisma#24244] [Q&A]
### Question

 this is the error I get:

**Raw query failed. Code: `unknown`. Message: `Kind: Command failed: Error code 115 (CommandNotSupported): let not supported, labels: {}**

which I understand that is due to using Azure Cosmo DB which got this limitation and it not fully compliant with Mongo DB API (v6.0):

The $lookup aggregation does not yet support the [uncorrelated subqueries](https://docs.mongodb.com/manual/reference/operator/aggregation/lookup/#join-conditions-and-uncorrelated-sub-queries) feature that's introduced in server version 3.6. If you attempt to use the $lookup operator with the let and pipeline fields, an error message that indicates that let is not supported appears.

What I am interested is there a workaround? Another way I can rewrite the code and/or the model in order to achieve the same
Thank you 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
model Response {
id String @id @default(auto()) @Map("_id") @db.ObjectId
updatedAt DateTime @updatedAt
responderId String @db.ObjectId
responder User @relation(fields: [responderId], references: [id], name: "Responder")
responseType ResponseType

requestId String @db.ObjectId
collabRequest CollabRequest @relation(fields: [requestId], references: [id])

seenByRequester Boolean @default(false)
}

model CollabRequest {
id String @id @default(auto()) @Map("_id") @db.ObjectId
requesterId String @db.ObjectId
requester User @relation(fields: [requesterId], references: [id], name: "Requester")
 ......
}
```

```ts
// Add any relevant Prisma Client queries here
     responses = await prisma.response.findMany({
        where: { collabRequest: {requesterId: req.query.requesterId}},
        orderBy: {updatedAt: "desc"},
```

- OS: 
- Database:  Cosmo DB (Mongo DB v6 )
- Node.js version: 
20.11.1
- Run `prisma -v` to see your Prisma version and paste it
5.12.0

**Comment by ludralph** [comment]
Hi @kharalanova 👋 

Currently, we do not have a workaround or full support for CosmosDB yet. You can take a look at the the open [issue](https://github.com/prisma/prisma/issues/2713).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
